src/bazel_version.rs

```diff
@@ -89,16 +89,16 @@ pub fn determine_bazelisk_version(path: &Path) -> Option<String> {
         return Some(version_str.trim().to_string());
     }
     let workspace_root = get_workspace_path(path)?;
-    if let Ok(bazeliskrc) = fs::read_to_string(workspace_root.join(".bazeliskrc")) {
+    if let Ok(bazelversion) = fs::read_to_string(workspace_root.join(".bazelversion")) {
```

Comment:
> When both of these files exist we should prefer the .bazelversion over the version in .bazeliskrc

Do you have a reference we could cite here? E.g., a permalink pointing into bazelisk's source code or something similar?

Would be great to also add that reasoning and the citation to the source code as a comment

Reply:
I updated the code to match Bazelisk according to: https://github.com/bazelbuild/bazelisk/blob/1f9a1aca958cdb50b4adb84b15cdda55a600ed31/README.md?plain=1#L45-L47